letsdrill.rb

```diff
@@ -1,24 +1,30 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  letter = 'F'
+  if integer >= 90
+    letter = 'A'
+  elsif integer >= 80
+    letter = 'B'
+  elsif integer >= 70
+    letter = 'C'
+  elsif integer >= 60
+    letter = 'D'
+  end
+  letter
```

Comment:
You don't really need a local variable here. The return value of the method is the value of the last statement in the method (as you know), and the return value of an `if`..`elsif`..`else`..`end` is the value of the evaluated clause.

This method returns `42`:

```ruby
def forty_two
  if 1 > 0
    42
  else
    :dafuq
  end
end
```